web/src/engine/predictive-text/worker-main/src/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+  "extends": "../../../../../tsconfig.base.json",
+
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "../build/obj",
+    "tsBuildInfoFile": "../build/obj/tsconfig.tsbuildinfo",
+    "rootDir": "./"
+  },
+  "references": [
+    { "path": "../../../../../../common/web/utils" },
+    { "path": "../../../../../../common/models/types"},
+    { "path": "../../../../../../common/models/wordbreakers"},
+    { "path": "../../../../../../common/web/lm-message-types"},
+    { "path": "../../../../../../common/models/templates"}
```

```suggestion
    { "path": "../../../../../../common/models/types"},
    { "path": "../../../../../../common/web/lm-message-types"}
```

While it makes sense on the surface, note that those projects are all worker-internal.  `worker-main` here is the worker-external interface with the worker, thus does not actually interact directly with wordbreaker or model-template types.

`Suggestion` and `Transform` are declared within `common/models/types`, so that one's fine, and the `lm-message-types` are about the interface between main window thread and worker thread, so those can stay.

I realize the entries were there before, but we really should be fine without them here now that we've noticed.